docs/configuration/sim-configuration.md

```diff
@@ -9,7 +9,11 @@ sim:
   enabled: true
 ```
 
-Also, you need to specify SignalFx token and realm, so at the end sim element in `values.yaml` looks like this:
+## Token and realm
+
+You need to specify Splunk Observability Cloud token and realm. There are two ways of configuring them:
+
+1. Pass those in a plain text via `values.yaml` so at the end sim element in looks like this:
```

Comment:
Is that 'in' in this part 'so at the end sim element in looks like this' necessary?

Reply:
good catch!